core/ledger/kvledger/txmgmt/rwsetutil/rwset_builder.go

```diff
@@ -116,6 +116,12 @@ func (b *RWSetBuilder) AddToPvtAndHashedWriteSet(ns string, coll string, key str
 	b.getOrCreateCollHashedRwBuilder(ns, coll).writeMap[key] = kvWriteHash
 }
 
+// AddToHashedWriteSetPurge adds a purge key to the hashed write-set
+func (b *RWSetBuilder) AddToHashedWriteSetPurge(ns string, coll string, key string) {
+	kvWriteHashPurge := newKVWriteHashPurge(key)
+	b.getOrCreateCollHashedRwBuilder(ns, coll).writeMap[key] = kvWriteHashPurge
```

Comment:
My understanding is that since the purge key is only being written to the hashed read write set that it would not require a call to getOrCreateCollPvtRwBuilder()